fuse-products/src/main/java/software/tnb/product/application/App.java

```diff
@@ -49,6 +49,10 @@ private void ensureDirNotPresent() {
         }
     }
 
+    protected boolean isAbleToRun() {
```

Comment:
`shouldRun` is probably a better name, I assume the app is always `able` to run :grin:

Reply:
you are right, thanks!